data.h

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <cstdio.h>
+#include <cstdlib.h>
+
+#include "cryptopp/sha.h"
+#include "cryptopp/Integer.h"
+
+class Data {
```

Comment:
For crypto data we're using either `std::string` or [`blob_t`](https://github.com/hyperledger/iroha/blob/develop/libs/common/types.hpp#L57)
What're the major features of this class that cannot be done via listed ones?